packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js

```diff
@@ -125,11 +139,15 @@ function ProjectHomePage ({
 
 ProjectHomePage.defaultProps = {
   inBeta: false,
+  organization: {},
   workflows: []
 }
```

Comment:
These should be moved into the `function` declaration. `defaultProps` is deprecated for functional components.